src/components/FeedbackInline/FeedbackInline.styles.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import { spacing, constants, typography } from "../../theme";
+import getThemeValue from "../../utils/getThemeValue";
+import { LinkCta, Text } from "../Text";
+
+export const Link = styled(LinkCta)`
+  color: ${getThemeValue("primary", "base")};
+
+  &:visited {
+    color: ${getThemeValue("primary", "base")};
```

Comment:
I think we should remove the darker visited link color for the `LinkCta` component. I will confirm with the design team.